qutebrowser/mainwindow/tabwidget.py

```diff
@@ -405,6 +408,9 @@ def __init__(self, win_id, parent=None):
         self.setAutoFillBackground(True)
         self.drag_in_progress = False
         stylesheet.set_register(self)
+        self.ensurePolished()
```

Comment:
Why is this needed?

Reply:
From what I understand, Qt does not immediately update `self.fontMetrics()` and other properties on a stylesheet change. `self.ensurePolished()` forces the update. Otherwise changes in the config are not reflected in real-time in terms of the icon size (which uses `fontMetrics()`). See https://doc.qt.io/qt-5/qwidget.html#ensurePolished as well as https://www.qtcentre.org/threads/55165-Howto-get-font-of-widget-set-by-stylesheet?p=282579#post282579